src/node_file.cc

```diff
@@ -825,7 +826,6 @@ void AfterMkdirp(uv_fs_t* req) {
     std::string first_path(req_wrap->continuation_data()->first_path());
     if (first_path.empty())
       return req_wrap->Resolve(Undefined(req_wrap->env()->isolate()));
-    node::url::FromNamespacedPath(&first_path);
```

Comment:
Is it expected?

Reply:
Yes because this was incomplete previously. In sync vs async call, the return value was different. Right now both values return the same thing.